lib/dialects/abstract/connection-manager.js

```diff
@@ -52,7 +52,7 @@ class ConnectionManager {
   onProcessExit() {
     if (this.pool) {
       this.pool.drain(() => {
-        Utils.getLogger().debug(`connection drain via pool due to process exit`);
+        Utils.debug(`connection drain via pool due to process exit`);
```

Comment:
debug should ideally be initiated by file so you have prefix like `sequelize:connection` or `sequelize:model`, atleast normally - however ideally our prefixed would be named for model names here.

I'd be fine with `const = require('Utils').debug()` even, i just hate seeing Utils prefixed scattered around the code :)